File: rest/subaccounts/exchanging-numbers-example-1/exchanging-numbers-example-1.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "{{ account_sid }}"; 
$token = "{{ auth_token }}"; 
$client = new Services_Twilio($sid, $token);

// Get an object from its sid. If you do not have a sid,
// check out the list resource examples on this page
$number = $client->accounts->get("{{ account_sid }}")
    ->incoming_phone_numbers->get("PN2a0747eba6abf96b7e3c3ff0b4530f6e");
$number->update(array(
    "AccountSid" => "{{ account_sid }}",
));
echo $number->phone_number;

File: rest/subaccounts/exchanging-numbers-example-1/exchanging-numbers-example-1.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "{{ account_sid }}";
    string AuthToken = "{{ auth_token }}";
    string CurrentAccount = "{{ account_sid }}";
    var twilio = new TwilioRestClient(AccountSid, AuthToken, CurrentAccount);

    var options = new PhoneNumberOptions();
    options.AccountSid = "{{ account_sid }}";
    
    twilio.UpdateIncomingPhoneNumber("PN2a0747eba6abf96b7e3c3ff0b4530f6e", options);
  }
}